### ISSUE TITLE
[Help]: Can you share training details about tta autoencoderKL module?

### ISSUE THREAD

**Issue by Jiang-Stan** [open-mmlab/Amphion#184]
## Problem Overview
(Briefly and clearly describe the issue you're facing and seeking help with.)
I tried to reproduce the TTA recipe but I failed.

## Steps Taken
(Detail your attempts to resolve the issue, including any relevant steps or processes.)
1. Preprocessed mel from AudioCaps is downloaded directly.
2. Download pretrained ckpt from [huggingface](https://huggingface.co/spaces/amphion/Text-to-Audio/tree/main/ckpts/tta), and write an inference script that can successfully get good wav from origin_mel and mel_out of AutoencoderKL
3. failed to train my own AutoencoderKL.

## Expected Outcome
(A clear and concise description of what you expected to happen.)
AutoencoderKL trained by my own performs the same as provided ckpt.

Three more questions, sincerely looking forward to your reply:
1. Did you train [step-0445000_loss-0.3306.pt](https://huggingface.co/spaces/amphion/Text-to-Audio/blob/main/ckpts/tta/autoencoder_kl_debug/checkpoints/step-0445000_loss-0.3306.pt) with only AudioCaps dataset?
2. Is the provided exp_config.json the same as your trained one?
3. The loss of my model dropped to 0.29 but it still performs bad. Is there any config I should taking care of during training and inference?
## Additional context
(Add any other context about the problem here.)

Thanks in advance for your sicerely help!

**Comment by Jiang-Stan** [comment]
Can you share training detail of TTA autoencoderKL? I tried to reproduce the result with the origin repo with your preprocessed mel data and training configs, but I failed to get result similar to opensource ckpt.

**Comment by HeCheng0625** [comment]
Hi, the detail can be found in https://github.com/open-mmlab/Amphion/blob/main/egs/tta/autoencoderkl/exp_config_latent_4_10_78.json

**Comment by fpicetti** [comment]
So the rest of the parameters are inherited from config/autoencoderkl.json, right?
